Composer/packages/client/src/components/BotRuntimeController/ErrorCallout.tsx

```diff
@@ -144,44 +137,25 @@ export const ErrorCallout: React.FC<IErrorCalloutProps> = (props) => {
   };
 
   return (
-    <Callout
-      setInitialFocus
-      ariaLabel={formatMessage(`{title}. {msg}`, { title: error.title, msg: error.message })}
-      data-testid={'errorCallout'}
-      gapSpace={0}
-      hidden={!visible}
-      target={target}
-      onDismiss={onDismiss}
-    >
-      <div css={calloutContainer}>
-        <p css={calloutLabel} id="callout-label-id">
-          {error.title}
-        </p>
-        <p css={calloutDescription} id="callout-description-id">
-          {buildErrorMessage(error)}
-          {error.linkAfterMessage != null && (
-            <Link href={error.linkAfterMessage.url} target={'_blank'}>
-              {error.linkAfterMessage.text}
-            </Link>
-          )}
-        </p>
-        {error.link != null && (
-          <p css={calloutLink}>
-            <Link href={error.link.url} target={'_blank'}>
-              {error.link.text}
-            </Link>
-          </p>
+    <div css={calloutContainer} data-testid="errorCallout">
+      <p css={calloutLabel} id="callout-label-id">
```

Comment:
this is not a very safe id, make sure it's "more" unique
If it's being used in another location, define the value in a variable instead

Reply:
Its actually unused. Removing them